src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,12 @@
+package seedu.addressbook.data.person;
+
+/**
+ * 
+ * new interface Printable
```

Comment:
errr... can you make the the header comment little more useful than "new interface ..."?